### PR TITLE
TC04

### PR DESCRIPTION
**Pull request by trangnhomanhcoi2601**

### CODE REVIEW

File: src/main/java/page_objects/BasePage.java [hunk 1]
```diff
@@ -8,14 +8,17 @@
 
     //Locators
     private final By tabLogin = By.cssSelector("li a[href='/Account/Login.cshtml']");
+    private final By ContactTab = By.cssSelector("li a[href='/Page/Contact.cshtml']");
```
Comment (MinhQuan119) [inline src/main/java/page_objects/BasePage.java]: It should be tabContact
Please check format anytime you create a new variables or method, make sure they are the same

Comment (trangnhomanhcoi2601) [reply src/main/java/page_objects/BasePage.java]: I've already fixed it. Please help me to check, thank so much.

File: src/test/java/tests/ContactTests.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import page_objects.ContactPage;
+
+public class ContactTests extends BaseTests {
+
+    ContactPage contactPage = new ContactPage();
+
+    @Test(description = "Contact Email contains correct href value which can help to quickly open Outlook Compose Message dialog")
+    public void tc04_CheckValueOfContactEmail() {
+        LogHelper.info("Go to contact tab");
+        contactPage.goToContactTab();
```
Comment (MinhQuan119) [inline src/test/java/tests/ContactTests.java]: At beforeMethod, you will stay at homePage by navigating URL. This row is the first action after that.
So it should be `homePage.goToContactTab();`

The same for preCondition in LoginTests 
It should be `homePage.goToLoginTab();`

Comment (MinhQuan119) [reply src/test/java/tests/ContactTests.java]: As Tan Le just said, the page name is correct if you don't have HomePage class
just need to update the method name

Comment (trangnhomanhcoi2601) [reply src/test/java/tests/ContactTests.java]: I've just changed method name. Please help me to check it. Thanks

**Review by MinhQuan119 — APPROVED** [review]

**Review by tanmle — APPROVED** [review]

**Pull request merged**